### DISCUSSION TITLE
broker load  hive map type  How to import

### DISCUSSION THREAD

**Discussion by pushiwuhua7** [apache/doris#6621] [A - General / Q&A] [answered]
COLUMNS TERMINATED BY "\\x01"
doris use varchar to accept hive map type
but errorMsg:type:ETL_QUALITY_UNSATISFIED; msg:quality not good enough to cancel
How do I import 😭

**Comment by EmmyMiao87** [comment, marked as answer]
First open the url to see a sample of the error data, which will write the specific reason. Doris does not currently have a map type, so he will treat map as a varchar.

**Reply by pushiwuhua7** [reply]
谢谢~
